src/components/mixins/DataFormat.js

```diff
@@ -35,7 +35,7 @@ export default {
     },
     formatValue(value) {
       if (!value && this.dataFormat !== 'boolean') {
-        return '';
+        return value;
```

Comment:
What are we checking for here? I feel like it'll error out if 'value' is bad. How is it used?

Reply:
Execution will only reach this part of the code if it's falsey and not a boolean. Before, if the value happened to be `null` or `undefined`, it would convert that value to an empty string and emit a change event; but this was causing a regression in modeler.

It seems @caleeli wanted to ensure that when no value has been entered by a user, it should assume the value is an empty string. But there's a few issues with this:
* It will emit a change event that essentially says "the value has been updated to an empty string" even if the user has not interacted with the input, as described above.
* It will emit this empty string value even if the data type is set to integer, for example, and clearly an empty string is not a valid integer.